packages/app/src/OAuthPage.tsx

```diff
@@ -1,20 +1,58 @@
 import { Title } from '@mantine/core';
-import { CodeChallengeMethod } from '@medplum/core';
-import { Logo, SignInForm } from '@medplum/react';
+import { CodeChallengeMethod, normalizeErrorString } from '@medplum/core';
+import { Logo, SignInForm, useMedplum } from '@medplum/react';
 import { useNavigate, useSearchParams } from 'react-router-dom';
 import { getConfig } from './config';
+import { useEffect, useState } from 'react';
+import { showNotification } from '@mantine/notifications';
+
+interface LogoInfo {
+  contentType: string;
+  url: string;
+  title: string;
+}
+
+interface ClientInfo {
```

Comment:
I think we can replace these types with the new auto-generated types in `ClientApplication.d.ts`.  This is basically the same as `ClientApplicationSignInForm`

```ts
import { ClientApplicationSignInForm } from '@medplum/fhirtypes';
```

That will be less code, and will also strongly enforce that they stay in sync with each other.